Sources/Spezi/Spezi.docc/Spezi_Ecosystem.md

```diff
@@ -0,0 +1,180 @@
+# The Spezi Ecosystem
+
+<!--
+This source file is part of the Stanford Spezi open-source project
+SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+SPDX-License-Identifier: MIT
+-->
+
+Discover the core modules available in the Spezi framework and understand their use cases.
+
+## Overview
+
+The Spezi ecosystem provides a collection of interoperable modules designed to accelerate healthcare and research application development. Each module is built to work independently or in combination with others, allowing you to choose the components that best fit your needs.
+
+## Core Modules
+Some of the following modules can be tested and interact with using the Template Application: **[SpeziTemplate](https://github.com/StanfordSpezi/SpeziTemplateApplication)**
```

Comment:
I would generally suggest embedding links right in the text & flow of the documentation rather then explicitly spelling them out:
```suggestion
Some of the following modules can be tested and interact with using the **[Spezi Template Application](https://github.com/StanfordSpezi/SpeziTemplateApplication)**
```